Sources/NostrSDK/Events/Calendars/CalendarEventRSVPFreebusy.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CalendarEventRSVPFreebusy.swift
+//
+//
+//  Created by Terry Yiu on 12/17/23.
+//
+
+import Foundation
+
+/// Determines if the user would be free or busy for the duration of the calendar event.
```

Comment:
Again, just a wording suggestion.
```suggestion
/// A type for describing whether the user is free or busy for the duration of the calendar event.
```